ide/xml.text/apichanges.xml

```diff
@@ -83,70 +83,6 @@ is the proper place.
     <!-- ACTUAL CHANGES BEGIN HERE: -->
 
     <changes>
-        <change id="deprecated-syntax-api">
-            <summary>Syntax API deprecated</summary>
-            <version major="2" minor="60"/>
-            <date day="18" month="10" year="2016"/>
-            <author login="sdedic"/>
-            <compatibility binary="compatible" source="incompatible" semantic="compatible" addition="yes" deprecation="yes" deletion="yes"/>
-            <description>
-                <p>
-                    The xml.text.syntax API uses the old and long deprecated lexer. The API is now moved to a new compatibility-only 
-                    <code>xml.text.deprecated90</code> module and a new API in <code>org.netbeans.modules.xml.text.api.dom</code> package is offered.               
-                </p>
-                <p>
-                    The following classes/packages have been moved to xml.text.obsolete90 module:
-                </p>
-                <ul>
-                    <li>
-                        <code>org.netbeans.modules.xml.text.syntax</code>
-                        <ul>
-                            <li>DTDSyntaxTokenMapper</li>
-                            <li>DTDTokenContext</li>
-                            <li>SyntaxElement</li>
-                            <li>UnicodeClasses</li>
-                            <li>XMLDefaultSyntax</li>
-                            <li>XMLSyntaxSupport</li>
-                            <li>XMLSyntaxTokenMapper</li>
-                            <li>XMLTokenIDs</li>
-                        </ul>
-                    </li>
-                    <li>
```

Comment:
Hi, we could also keep that for history and add a link to this PR in the description https://github.com/apache/netbeans/pull/4821 to keep track

Reply:
Hey @ebarboni , I really like that suggestion. Done. 

How would one go about getting the changes to the build server so that the test doesn't get called?